OSpackages.sh

```diff
@@ -163,12 +165,19 @@ function PrepChroot {
    local -a BASEPKGS
 
    # Create an array of packages to install
-   mapfile -t BASEPKGS < <(
-      rpm --qf '%{name}\n' -qf /etc/os-release ; \
-      rpm --qf '%{name}\n' -qf  /etc/yum.repos.d/* 2>&1 | grep -v "not owned" | sort -u ; \
-      echo yum-utils
+   BASEPKGS=(
+      yum-utils
    )
 
+   # Don't try to be helpful if doing cross-distro (i.e., "bootstrapper-build")
+   if [[ -z ${ISCROSSDISTRO} ]]
```

Comment:
Does this need a default assignment to support `set -u`?

```suggestion
   if [[ -z ${ISCROSSDISTRO:-} ]]
```

Reply:
I ran it past `shellcheck` and it didn't bark, but, probably run into if were to enable full strict-mode?

Reply:
Yeah, I think by default shellcheck evaluates strictly only if the related options are set in the shebang or at the top of the file.